restricted-site-access.php

```diff
@@ -1221,6 +1270,41 @@ public static function is_network( $plugin ) {
 
 	}
 
+	/**
+	 * Dialog markup to warn network-wide RSA disable
+	 *
+	 * @return void
+	 */
+	public static function admin_footer() {
+		$current_screen = get_current_screen();
+
+		if ( 'plugins-network' !== $current_screen->id ) {
+			return;
+		}
+		?>
+		<div id="rsa-disable-dialog" class="hidden">
+			<h2><?php esc_html_e( 'Confirm network deactivation', 'restricted-site-access' ); ?></h2>
+			<p><?php esc_html_e( 'You are about to network disable the Restricted Site Access plugin making all sites on this network publicly available.', 'restricted-site-access' ); ?></p>
+			<p>
+				<?php
+				printf(
+					/* translators: %s: The words 'I understand'. */
+					esc_html__( 'If you are sure about your action, please type %s to proceed.', 'restricted-site-access' ),
```

Comment:
This is going to escape the `<code>` tag, no?

Reply:
yes, this should be wp_kses_post

Reply:
Right! Fixed in https://github.com/10up/restricted-site-access/pull/29/commits/866fe5b159bb00ff7863f96199c3e1d3b7a5df01